whiteboardflow-frontend/src/components/Whiteboard/Whiteboard.js

```diff
@@ -50,12 +53,13 @@ const Whiteboard = () => {
                             minWidth: 1240,
                             minHeight: 2000,
                             guides: {
-                                enable: true,
-                                gap: 50,
-                                type: 'line'
-                            }
+                                enable: false
+                            },
                         },
                         grabber: {
+                            listenerOptions:{
+                                capture: true,
```

Comment:
This breaks mouse drawing

Reply:
I know, but do we need mouse drawing, I'm actually trying to get rid of both mouse or touch drawing